recipe/meta.yaml

```
@@ -20,26 +20,28 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - make      # [not win]
+    - make # [not win]
     - libtool   # [not win]
     - patch       # [not win]
     - m2-patch    # [win]
   host:
-    - libiconv  # [win]
-    - make      # [not win]
+    - libiconv
+    - minizip
+    - expat
```

```suggestion
    - expat {{ expat }}
```
Because of https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L70